src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -696,6 +700,10 @@ int goto_instrument_parse_optionst::doit()
       status() << "Performing full inlining" << eom;
       goto_inline(goto_functions, ns, ui_message_handler);
 
+      status() << "Removing calls to functions without a body" << eom;
+      remove_calls_no_bodyt remove_calls_no_body;
+      remove_calls_no_body(goto_functions);
+
```

Comment:
Why is this specifically done here - is this the only place calling `goto_inline`?